manifests/s/ScreamingFrog/SEOSpider/16.4/ScreamingFrog.SEOSpider.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ScreamingFrog.SEOSpider
+PackageVersion: "16.4"
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.screamingfrog.co.uk/products/seo-spider/ScreamingFrogSEOSpider-16.4.exe
+  InstallerSha256: 1c4fdc3eec5220e2456555fe4e0f3002996199b3f292bd63ba6ff6275525e150
+ManifestType: installer
+ManifestVersion: 1.2.0
```

```suggestion
ManifestVersion: 1.1.0
```
1.2.0 doesn't exist at the moment.

If you're making an update for this package, only PackageVersion, URL, and Hash needs to be changed.